.github/workflows/integrations.yml

```diff
@@ -0,0 +1,76 @@
+
+name: RMT Integration tests
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 3
+      matrix:
+        ruby-version: ['2.5.8', '2.6.6', '2.7.2']
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_DATABASE: rmt_test
+          MYSQL_USER: rmt
+          MYSQL_PASSWORD: rmt
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306:3306
+        options: >-
+          --health-cmd="mysqladmin ping"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=3
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '100'
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+          bundler-cache: true
+          bundler: 1.17
+
+      - name: "Cache gems"
+        uses: actions/cache@v2
+        id: gem-cache
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - name: Pre-run inline scripts
+        run: |
+          sudo mkdir -p /var/lib/rmt
+          ruby -e "require 'yaml'; puts({'database_test'=>{'host' => '127.0.0.1', 'port' => 3306, 'username'=>'rmt','password'=>'rmt','database'=>'rmt_test','adapter'=>'mysql2','encoding'=>'utf8','timeout'=>5000,'pool'=>5}}.to_yaml)" > config/rmt.local.yml
+
+      - name: Run tests
+        env:
+          RAILS_ENV: test
+        shell: 'script -q -e -c "bash --noprofile --norc -eo pipefail {0}"'
```

Comment:
I'd try to get rid of this, since it was added out of desperation and I don't think we really need it. Let's remove unnecessary complexity...